tests/test_dataclasses.py

```diff
@@ -199,21 +205,26 @@ class TestOffer:
     def setup_method(self):
         self.initial_data = {
             "id": uuid.uuid4(),
-            "creation_time": DateTime.now(),
-            "price": 10,
+            "creation_time": test_time_stamp(),
+            "price": 10.,
             "energy": 30,
             "original_price": 8,
             "attributes": {},
             "requirements": [],
-            "seller": "seller"
+            "seller": "seller",
+            "time_slot": test_time_stamp()
         }
+        print(self.initial_data)
```

Comment:
reminder to remove, same applies to the rest of print statements across this file

Reply:
Weird, seems that i forgot to review this file before pushing. Will do.